index.jsx

```diff
@@ -168,13 +168,14 @@ module.exports = function Dimensions ({
         }
         return (
           <div style={wrapperStyle} ref='wrapper'>
-            {(containerWidth || containerHeight) &&
-              <ComposedComponent
+            {(containerWidth || containerHeight)
```

Comment:
`ComposedComponent` is already not being created. `{(0 || 0) && <ComposedComponent>}` becomes `{0}` which will render "0" as text. `{(0 || 0) ? <ComposedComponent> : null}` become `{null}` which will render nothing.